kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/JbpmKieServerExtension.java

```diff
@@ -496,6 +497,9 @@ public boolean isUpdateContainerAllowed(String id, KieContainerInstance kieConta
     public void updateContainer(String id, KieContainerInstance kieContainerInstance, Map<String, Object> parameters) {
         // essentially it's a redeploy to make sure all components are up to date,
         // though update of kie base is done only once on kie server level and KieContainer is reused across all extensions
+        // we add this UPDATING_CONTAINER_PARAM
+        parameters.put(UPDATING_CONTAINER_PARAM, Boolean.TRUE);
```

Comment:
Maybe use such parameter more for the dispose container so the condition would abortInstances or dispose. That way it’s more readable in my opinion

Reply:
@mswiderski you're right, maybe it is not clear enough.. Better now?